orca-keel/src/test/kotlin/com/netflix/spinnaker/orca/keel/ImportDeliveryConfigTaskTests.kt

```diff
@@ -377,26 +379,32 @@ internal class ImportDeliveryConfigTaskTests : JUnit5Minutests {
                 manifestLocation.manifest,
                 manifestLocation.ref
               )
-            } throws RetrofitError.httpError(
+            } returns manifest
+          }
+          with(keelService){
+            every {
+              publishDeliveryConfig(manifest)
+            } throws SpinnakerHttpException(RetrofitError.httpError(
               "http://keel",
               Response(
                 "http://keel", 403, "", emptyList(),
-                JacksonConverter(objectMapper).toBody(accessDeniedError) as TypedInput
+               converter.toBody(accessDeniedError) as TypedInput
               ),
-              null, null
-            )
+              converter, null
+            ))
           }
         }
 
         test("task fails and includes the error details returned by keel") {
           val result = execute(manifestLocation.toMap())
           expectThat(result.status).isEqualTo(ExecutionStatus.TERMINAL)
-          expectThat(result.context["error"]).isA<SpringHttpError>().isEqualTo(accessDeniedError)
+          expectThat(result.context["error"]).isA<SpringHttpError>()
```

Comment:
Can we restore an assertion on the value of the error here?  It's a shame to remove it and weaken the test.  I tried
```suggestion
          **expectThat(result.context["error"]).isA<SpringHttpError>().isEqualTo(accessDeniedError)
```
but it fails with:
```
      ✗ is equal to SpringHttpError(error=Forbidden, status=403, message=Access denied, timestamp=2024-02-15T16:50:02.144Z, details=null)
              found SpringHttpError(error=Forbidden, status=403, message=Access denied, timestamp=2024-02-15T16:50:02.165917Z, details=null)
```
is there something different we can do with the timestamp so that passes?

Reply:
I have moved this part of the test to java file , which uses wiremock. This is covered in the commit : 7a48b311ff9454124496432a02fe12e1db9aae8e